packages/sensors_plus/sensors_plus/example/integration_test/sensors_plus_test.dart

```diff
@@ -22,4 +23,68 @@ void main() {
     });
     expect(await completer.future, isNotNull);
   });
+
+  testWidgets('Can subscribe to accelerometerEvents and get expected events',
+      (WidgetTester tester) async {
+    final completer = Completer<AccelerometerEvent>();
+    StreamSubscription<AccelerometerEvent> subscription;
+    subscription = accelerometerEvents.listen((AccelerometerEvent event) {
+      completer.complete(event);
+      subscription.cancel();
+    });
+
+    final event = await completer.future;
+    expect(
+        await event.toString(),
+        AccelerometerEvent(0.0, 9.776321411132812, 0.812345027923584)
+            .toString());
+  }, skip: !Platform.isAndroid);
+
+  testWidgets('Can subscribe to gyroscopeEvents and get non-null events',
+      (WidgetTester tester) async {
+    final completer = Completer<GyroscopeEvent>();
+    StreamSubscription<GyroscopeEvent> subscription;
+    subscription = gyroscopeEvents.listen((GyroscopeEvent event) {
+      completer.complete(event);
+      subscription.cancel();
+    });
+    expect(await completer.future, isNotNull);
+  });
+
+  testWidgets('Can subscribe to gyroscopeEvents and get expected events',
+      (WidgetTester tester) async {
+    final completer = Completer<GyroscopeEvent>();
+    StreamSubscription<GyroscopeEvent> subscription;
+    subscription = gyroscopeEvents.listen((GyroscopeEvent event) {
+      completer.complete(event);
+      subscription.cancel();
+    });
+
+    final event = await completer.future;
+    expect(event.toString(), GyroscopeEvent(0, 0, 0).toString());
+  }, skip: !Platform.isAndroid);
+
+  testWidgets(
+      'Can subscribe to userAccelerometerEvents and get non-null events',
+      (WidgetTester tester) async {
+    final completer = Completer<UserAccelerometerEvent>();
+    StreamSubscription<UserAccelerometerEvent> subscription;
+    subscription =
+        userAccelerometerEvents.listen((UserAccelerometerEvent event) {
+      completer.complete(event);
+      subscription.cancel();
+    });
+    expect(await completer.future, isNotNull);
+  });
```

Comment:
This test is repeated, please delete

Reply:
Our plugin seems to have four functions, Accelerometer, UserAccelerometer, Gyroscope, and Magnetometer. are we sure we want to remove the UserAccelerometer test?

<img width="393" alt="image" src="https://user-images.githubusercontent.com/43690894/188340410-e46bccfb-b3cc-43e4-a794-54b488150a4a.png">

Reply:
The test is duplicated, see the file, there are two tests that do exactly the same.

Line 16 and line 69.

Reply:
Yes, they are two different methods